sdf/gearSDF.glsl

```diff
@@ -0,0 +1,28 @@
+#include "../math/map.glsl"
+
+/*
+original_author: Kathy McGuiness
+description: |
+    Returns a gear shaped SDF
+    Some notes about the parameters:
+        * b determines the length and roundness of the spokes
+        * n is the number of spokes 
+use: gearSDF(<vec2> st, <float> b, <int> n_spokes)
+*/
+
+#ifndef FNC_GEARSDF
+#define FNC_GEARSDF
+float hyperbolicTan( float theta ) {
```

Comment:
Would be possible to incorporate this to gearSDF?

Reply:
Yes, I was able to incorporate.